samples/net/http_server/boards/nrf9160dk_nrf9160_ns.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration file for nRF9160 DK
+# This file is merged with prj.conf in the application folder, and options
+# set here will take precedence if they are present in both files.
+
+# General
+CONFIG_PICOLIBC=y
+
+# Disable Duplicate Address Detection (DAD)
+# due to not being properly implemented for offloaded interfaces.
+CONFIG_NET_IPV6_NBR_CACHE=n
+CONFIG_NET_IPV6_MLD=n
```

Comment:
Without these changes the sample hard faults. Initially discovered when the _modem_net_if_ module was developed which integrates with zephyr NET conn mgr API for LTE. Fixing this has probably been forgotten or down prioritized, it might be that @glarsennordic created a ticket for it. Easy to reproduce.

Reply:
I have some doubts if this PR will work properly with IPv6 networks if MLD is disabled. MLD is pretty much mandatory for IPv6. How well we have tested this?

Reply:
This is only disabled for cellular builds. It shouldn't "have any effect" as IP is offloaded and handled by the modem anyway. I haven't had any related issues with IPv6 + conn mgr and modem net if before.
At least not related to this in particular.

So perhaps this is more about making this work properly when the net interface is offloaded.

Reply:
If modem handles it (MLD) itself, then it should be ok. But for wifi we would need to enable it.

Reply:
Jepp, this is only for 91 builds since its in the 91 specific board file.

Reply:
@simensrostad No ticket that I am aware of -- as you mention, I don't think it's an actual bug, is it?

Reply:
Well, if you remove those all samples that use the modem net if + conn mgrf implementation hard faults. So depends on how you look at it. You shouldn't really be allowed to enable MLD (and likely other IP related modules as well) if the TCP/IP stack is offloaded.

Ok, I thought this was scheduled to be addressed. Ill see if I can locate the discussions we had regarding it and formulate a ticket. Thanks